src/boat_simulator/boat_simulator/nodes/physics_engine/physics_engine_node.py

```diff
@@ -182,6 +183,12 @@ def __init_private_attributes(self):
             generator=MVGaussianGenerator(current_mean, current_cov)
         )
 
+        # Used arbitrary non-zero stdev centred about 0 for now
+        # No delay in this instance
+        sim_wind = self.__wind_generator.next()
+        sim_wind_noise_stdev = List[Scalar] = [-10.0, 10.0]
```

Comment:
The default noise of `[1.0, 1.0]` in the `SimWindSensor` constructor seems reasonable for wind. Could you please adjust this to use the default.